src/wing/CCPACSWingRibRotation.cpp

```diff
@@ -31,7 +31,7 @@ CCPACSWingRibRotation::CCPACSWingRibRotation(CCPACSWingRibsPositioning* parent)
 void CCPACSWingRibRotation::SetZ(const double & value) {
     generated::CPACSRibRotation::SetZ(value);
     // invalidate whole component segment structure since rib may be referenced anywhere
-    m_parent->GetParent()->GetStructure().Invalidate();
+    m_parent->GetParent()->GetParent()->GetParent()->Invalidate();
```

Comment:
I would not call it a design flaw (the rib rotation really has to invalidate it's structure, which is a parent 4 steps above), but it is definitely not looking great^^